src/System.Management.Automation/engine/LanguagePrimitives.cs

```diff
@@ -22,9 +22,8 @@
 using System.Diagnostics.CodeAnalysis;
 using Dbg = System.Management.Automation.Diagnostics;
 using MethodCacheEntry = System.Management.Automation.DotNetAdapter.MethodCacheEntry;
-
-#if !CORECLR
-// System.DirectoryServices are not in CoreCLR
+#if !UNIX
+using System.Management;
 using System.DirectoryServices;
 #endif
```

Comment:
Can sort the using list?

Reply:
I'll put the two here in alpha order.  Won't touch the ones above.